### PR TITLE
Set up API routes for CRUD operations

### PR DESCRIPTION
**Pull request by timluu-zz**
-also set up changelog file

@andrew121212 @wizard-jS @Ceciil 
Please review the following PR - thanks!

### CODE REVIEW

**Review by anlee3 — DISMISSED** [review]
Your DELETE is missing information. You might want to consider writing what is expected on success and error responses.

File: README.md [hunk 1]
```diff
@@ -22,13 +22,157 @@ npm run seeding
 npm run react-dev
 npm run server
 
+### CRUD Operations
```
Comment (andrewgeorgemitchell) [inline README.md:25]: Nice Job this is very clear documentation

File: server/__temp__index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
+const express = require('express');
```
Comment (andrewgeorgemitchell) [inline server/__temp__index.js:1]: Good routes, consider exporting them to a controller file using express.Router()

**Review by andrewgeorgemitchell — APPROVED** [review]
Overall this is good! Made few comments mostly to do with improving separation of concerns and file system improvements

**Review by timluu-zz — COMMENTED** [review]
Thanks - I fixed it!

**Pull request merged**